forge/routes/api/project.js

```diff
@@ -820,7 +820,9 @@ module.exports = async function (app) {
      *  - returns most recent 30 entries
      *  - ?cursor= can be used to set the 'most recent log entry' to query from
      *  - ?limit= can be used to modify how many entries to return
+     * @name /api/v1/projects/:id/logs // << Added an 's'
      * @name /api/v1/projects/:id/log
+     * @name /api/v1/projects/:id/logs // << Added an 's'
```

Comment:
Should this be named three times?

Reply:
nope. that is a VSCode bug I was recording and reporting - it slipped by.

Reply:
fixed in [1c04d76](https://github.com/flowforge/flowforge/pull/2318/commits/1c04d76aed5b420d471914fec2f2528286d15135)